src/idiom275.fuzion

```diff
@@ -0,0 +1 @@
+array (s.byte_length / 8) i->s.substring((i*8),(i*8+8)).parse_integer(u8, 2)
```

Comment:
nice short solution! 
Add LFs and do not use `()` in calls:
```suggestion
array s.byte_length/8 i->
  s.substring i*8 i*8+8
   .parse_integer u8 2
```
can you also try this
```suggestion
array u8 s.byte_length/8 i->
  s.substring i*8 i*8+8
   .parse_integer 2
```
I think this does not work, but would be nice if type propagation worked in this direction as well. If this does not work, add an issue to the fuzion repo.